fbw-a380x/README.md

```diff
@@ -15,48 +15,48 @@ This is essentially a clean build.
 
 These systems are currently not operational / these features are missing and shall thus not lead to bug reports.
 
-### ATA 20 Aircraft General / Exterior 3D Model
+This list is divided into standardized ATA chapters, if there are no noteworthy systems missing from a chapter, the chapter heading is hidden (this was different in a previous version of this list). This list is not exhaustive, there might be details missing in other systems which are not stated here.
 
-### ATA 21 Air Con / Pressurization / Ventilation
+<!-- ### ATA 20 Aircraft General / Exterior 3D Model -->
 
-### ATA 22 Flight Envelope
+<!-- ### ATA 21 Air Con / Pressurization / Ventilation -->
 
-- PRIM FE part (FMS taking over atm)
+<!-- ### ATA 22 Flight Envelope -->
 
 ### ATA 22 Autoflight / FCU
 
-- AP tuning for A380X
 - FLS
 - THR NOISE, DCLB
 
 ### ATA 22 Flight Management System (FMS)
 
-- Incomplete pages see fbw-a380x/src/systems/instruments/src/MFD/pages
-- Separation into three synchronized FMCs and two FMSs (currently one FMC serving one FMS)
+- Incomplete pages see [fbw-a380x/src/systems/instruments/src/MFD/pages](https://github.com/flybywiresim/aircraft/tree/master/fbw-a380x/src/systems/instruments/src/MFD/pages)
+- Full separation into three synchronized FMCs and two FMSs (currently one FMC serving one FMS)
 - EO modes / EO SID
 - Accurate CI, OPT and REC MAX FL computation
 - NADP (see THR NOISE above)
 - Fuel planning on FMS/FUEL&LOAD page
-- ATC / datalink
+- ATC / datalink / CPDLC
 - FCU BKUP
 
 ### ATA 23 Communication
 
 - RMP: HF, TEL, MENU, SATCOM, DATALINK pages
 - CIDS: CALLS buttons
 
-### ATA 24 Electrical
+<!-- ### ATA 24 Electrical -->
 
-### ATA 25 Equipment / Cockpit 3D Model
+<!-- ### ATA 25 Equipment / Cockpit 3D Model -->
 
-### ATA 26 Fire and Smoke Protection
+<!-- ### ATA 26 Fire and Smoke Protection -->
 
 ### ATA 27 Flight Controls
 
-- Complete SLAT/FLAP CTL implementation (adapted A32NX computers atm)
-- Automatic pitch trim setting
+- Completely migrated SLAT/FLAP CTL implementation (adapted A32NX computers at the moment)
+- Automatic THS setting on ground
 
 ### ATA 28 Fuel
+
 - Refuel Driver/Refuel Application placeholder pseudo-FQMS implementation (pre-cursor for realistic FQMS implementation while supporting instant load)
   - CPIOM etc. not implemented yet.
 - Automatic fuel transfer system - Not fully implemented yet i.e. AGT off when 2 engines on
```

Comment:
This particular item has led someone to believe that automatic fuel transfers are not implemented at all. In any case, I believe it is unclear as it refers to an acronym that is not explained as part of only one case that doesn't fully explain what is not implemented vs what is.

I suggest something like - "Portions of the automatic fuel transfer system -- main transfers from the inner, mid, trim, and outer tanks are implemented with the correct scheduling as are CG control transfers from the trim tank to the appropriate inner, mid, or feed tanks. Not implemented yet are load alleviation transfers, transfers based on remaining time to destination, cold fuel transfers, and automatic transfers that occur on the ground."